peoplehr/tests/test.bal

```diff
@@ -24,7 +24,10 @@ configurable string baseURL = os:getEnv("BASE_URL");
 configurable string emailAddress = os:getEnv("EMAIL_ADDRESS");
 configurable string password = os:getEnv("PASSWORD");
 configurable string queryName = os:getEnv("QUERY_NAME");
-string employeeId = "test01";
+configurable int screenId = 380477;
+configurable string employeeId = "test01";
+int txnId = 0;
+int applicantId = 7696154;
```

Comment:
Should we not made these configurable as well?

Reply:
I guess no need as these are used by the test cases and initialized themselves.